langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -457,6 +463,12 @@ public function matches(string str, RegExp re) returns boolean {
 # Tests whether there is a match of a regular expression somewhere within a string.
 # This is equivalent to `regexp:find(re, str, startIndex) != ()`.
 #
+# ```ballerina
+# "Will Match Somewhere".includesMatch(re `A|Th.*ch|^`) ⇒ true
+#
+# "Not a match".includesMatch(re `A|Th.*ch`) ⇒ false
+# ```
```

Comment:
Don't we need to add an example with start index parameter?

Reply:
7559f0a